### PR TITLE
Tests/moving v.4.2.3 to unit tests

### PR DESCRIPTION
**Pull request by sigrunnu**
Moved issue test to unit test, v.4.2.3:
#4679
#4700
#4813
#4856
#5010
#5075

### CODE REVIEW

File: samples/unit-tests/pointer/click/demo.js [hunk 1]
```diff
@@ -1,3 +1,69 @@
+function test(type, tooltipEnabled) {
```
Comment (TorsteinHonsi) [inline samples/unit-tests/pointer/click/demo.js]: This is polluting the global scope, which may come back and haunt us later. Wrap the `test` function and its calls (only the changes in this diff) in an iife.

File: samples/unit-tests/series-bubble/minmaxsize/demo.js [hunk 2]
```diff
@@ -1,3 +1,52 @@
+QUnit.test('Setting X axis extremes on bubble (#5167)', function (assert) {
```
Comment (TorsteinHonsi) [inline samples/unit-tests/series-bubble/minmaxsize/demo.js]: Not related to `minSize` or `maxSize`, so a better location would be `unit-tests/series-bubble/extremes`.

File: samples/unit-tests/series-column/zero-column-visible/demo.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+QUnit.test('Zero column visible (#5146)', function (assert) {
```
Comment (TorsteinHonsi) [inline samples/unit-tests/series-column/zero-column-visible/demo.js]: The folder name is a bit too specific, rename it to `series-column/data`.

File: samples/unit-tests/tooltip/zindex/demo.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+function test(key, options) {
```
Comment (TorsteinHonsi) [inline samples/unit-tests/tooltip/zindex/demo.js]: Polluting the global namespace, wrap the whole content of this file in an iife or a QUnit.module.

**Comment by sigrunnu** [comment]
The two commits "renamed file" is one renaming and one deleting of the old. Both belongs to issue test #5146.

The "Wrapped #5146 in iife" is for issue test #4679 and not #5146.

**Pull request merged**